app/src/ui/tab-bar.tsx

```diff
@@ -30,11 +33,17 @@ export class TabBar extends React.Component<ITabBarProps, {}> {
   private readonly tabRefsByIndex = new Map<number, HTMLButtonElement>()
 
   public render() {
+    const { type } = this.props
+
     return (
       <div
         className={
           'tab-bar ' +
-          (this.props.type === TabBarType.Switch ? 'switch' : 'tabs')
+          (type === TabBarType.Switch
+            ? 'switch'
+            : type === TabBarType.Vertical
+            ? 'vertical'
+            : 'tabs')
```

Comment:
would be nice to pull this out into a little `switch...case` with an `assertNever`, but not a big deal. (non-blocking)